src/main/java/utils/Utils.java

```diff
@@ -109,6 +109,8 @@ public static void setObject(PreparedStatement stmt, int index, Object obj)
       stmt.setFloat(index, (Float)obj);
     } else if (obj instanceof Double) {
       stmt.setDouble(index, (Double)obj);
+    } else if (obj == null) {
+      throw new IllegalArgumentException("object is null");
```

Comment:
This is good, I like it. There's a discussion we could have about what to do in this case, but this is probably how it would've ended so I'm good with this.